src/compiler/crystal/codegen/codegen.cr

```diff
@@ -5,7 +5,7 @@ require "../semantic/type_inference"
 require "../program"
 require "./llvm_builder_helper"
 
-LLVM.init_x86
+LLVM.init_all_targets
```

Comment:
It may be interesting to look at the ARM port about that. Thought I guess we don't need it for FreeBSD as long as it runs on x86.